newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -94,48 +89,80 @@ export default class CommentEvent extends React.Component<
   };
 
   render() {
-    const commentEvent = gd.asCommentEvent(this.props.event);
-    const color = rgbToHex(
-      commentEvent.getBackgroundColorRed(),
-      commentEvent.getBackgroundColorGreen(),
-      commentEvent.getBackgroundColorBlue()
-    );
-    const textColor = rgbToHex(
-      commentEvent.getTextColorRed(),
-      commentEvent.getTextColorGreen(),
-      commentEvent.getTextColorBlue()
-    );
+    var commentEvent = gd.asCommentEvent(this.props.event);
+
+    const r = commentEvent.getBackgroundColorRed(),
+      g = commentEvent.getBackgroundColorGreen(),
+      b = commentEvent.getBackgroundColorBlue();
+
+    const textColors = {
```

Comment:
`textColor`

also, there is now a color using `red`, `green`, `blue` and another being just variable r,g,b.
Can we make this consistent? Is this useful compared to the rgbToHex that we had before?